tests/Database/DatabaseEloquentModelTest.php

```diff
@@ -447,6 +447,21 @@ public function testTimestampsAreCreatedFromStringsAndIntegers()
         $this->assertInstanceOf(Carbon::class, $model->created_at);
     }
 
+    public function testSerializeDate()
+    {
+        $model = new EloquentDateModelStub;
+        $model->created_at = '2018-11-05 09:26:00';
+        $this->assertEquals($model->attributesToArray()['created_at'], '2018-11-05 09:26:00');
+        Carbon::serializeUsing(function ($carbon) {
+            return intval($carbon->format('U'));
```

Comment:
I think the practice is to use short int cast `(int) $carbon->format('U')` in this project.

Reply:
Fixed